QUICKSTART.md

```diff
@@ -0,0 +1,251 @@
+# Quick Start Guide
+
+## Overview
+
+**Hypha** is an open-source tool for **<short purpose sentence>**. It helps you **<problem it solves / value statement>**.
+
+This guide will walk you through installation, configuration, and running your first **<project unit>**.
+
+---
+
+## Prerequisites
+
+Make sure you have the following installed:
+
+| Requirement | Version | Check Command       |
+| ----------- | ------- | ------------------- |
+| Rust        | ≥ 3.13  | `python3 --version` |
```

Comment:
Do we require Rust? If users downloads our binaries, they won't need Rust to run them.